CodeStarterCamp_Week2/main.swift

```diff
@@ -8,5 +8,59 @@
 
 import Foundation
 
-print("Hello, World!")
+let lottoCountLimit = 6
+let lottoNumberLimit = 45
+let lottoNumberRange = 1...lottoNumberLimit
 
+func generateRandomNumber() -> Int{
```

Comment:
```suggestion
func generateRandomNumber() -> Int {
```
아래 함수들은 잘 작성해주셨는데 두개의 함수가 컨벤션에 맞지 않네용 개선해보아요!!